include/mata/nfa.hh

```diff
@@ -574,7 +574,30 @@ public:
      */
     void get_one_letter_aut(Nfa& result) const;
 
-    void print_to_DOT(std::ostream &outputStream) const;
+    /**
+     * @brief Prints the automaton in DOT format
+     * 
+     * @return automaton in DOT format 
+     */
+    std::string print_to_DOT() const;
+    /**
+     * @brief Prints the automaton to the output stream in DOT format
+     */
+    void print_to_DOT(std::ostream &output) const;
+    /**
+     * @brief Prints the automaton in mata format
+     *
+     * If you need to parse the automaton again, use IntAlphabet in construct()
+     * 
+     * @return automaton in mata format 
+     */
+    std::string print_to_mata() const;
```

Comment:
Maybe, we could unify this with name of parsing function? I believe it is `parse_mf`, so maybe it could be `print_mf`/`dump_mf`/`write_mf` or something similar?

Reply:
Yeah, we should probably unify the names, but first we need to decide which one to use. Is `_mf` all right? Should we keep that?

Reply:
Well, now that you are saying maybe `_mata` is better, since the files have `.mata` extension :D Or change or files to `.mf`

Reply:
Then maybe we should decide how the mata file format extension should look first. I'm for `.mata`.

Reply:
Definitely `_mata`. And I though we already decided on `.mata` for file extension name.

Reply:
don't remember, .mata and _mata sounds better to me also.